src/widgets/arc/lv_arc.h

```diff
@@ -223,6 +223,13 @@ int16_t lv_arc_get_max_value(const lv_obj_t * obj);
  */
 lv_arc_mode_t lv_arc_get_mode(const lv_obj_t * obj);
 
+/**
+ * Get whether the arc is type or not.
+ * @param arc       pointer to an arc object
+ * @return          arc's current rotation
+ */
+int16_t lv_arc_get_rotation(const lv_obj_t * obj)
```

Comment:
Looks like there's a missing semicolon here. I think this is the reason for the CI failure.

Reply:
Updated